index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <h2 class="h2"><a>Welcome</a></h2>
+        <nav>
+            <img class="nav" src="./images/Union.png" alt="">
+        </nav>
+    </header>
+    <section class="intro">
+        <h1 class="h1">
+        Hey there. I’m <br>Richard
+        </h1>
+        <h2 class="sd">
+            I’m a software developer
+        </h2>
+            <p class="prg">
+                I can help you build a product ,
+                 feature or website Look through some of my work and experience!
+                  If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+            <ul class="icon-list">
+                <li><a href="https://twitter.com/richardfriday14"><img src="./images/twitter.png" alt="twitter"></a></li>
+                <li><a href="https://www.linkedin.com/in/richard-friday-54980718a"><img src="./images/Linkedin.png" alt="linkedin"></a></li>
+                <li><a><img src="./images/medium.png" alt="medium"></a></li>
+                <li><a href="https://github.com/Richez003"><img src="./images/github.png" alt="github"></a></li>
+                <li><a href="https://angel.co/jobs?ref=onboarding"><img src="./images/angellist.png" alt="angellist"></a></li>
+                </ul>
+                <span class="dis"> 
+                    <img src="./images/Disabled.png" alt="disabled">
+                </span>
```

Comment:
- [ OPTIONAL] Kindly make this down icon to be responsive for all mobile versions size :arrow_down: 

![fool](https://user-images.githubusercontent.com/57562869/159223799-53c2ab02-b201-491e-9522-7cb6e698a9ba.png)